tests/integrations/subprocess/test_Subprocess__Popen.py

```diff
@@ -36,9 +38,9 @@ def test_simple_call_with_path_arg(mock_sub, capsys, tmp_path):
 
 def test_simple_verbose_call(mock_sub, capsys):
     "If verbosity is turned out, there is output"
-    mock_sub.command.verbosity = 2
+    mock_sub.command.logger = Log(verbosity=2)
 
     mock_sub.Popen(['hello', 'world'])
 
     mock_sub._subprocess.Popen.assert_called_with(['hello', 'world'])
-    assert capsys.readouterr().out == ">>> hello world\n"
+    assert capsys.readouterr().out.splitlines()[-1] == ">>>     hello world"
```

Comment:
As with the other output test - this is a slightly different test, and the origin of the extra space isn't clear

Reply:
Updated the test to compare entire output.

Extra whitespace is from command being indented. Further explanation in other comment.